language/move-lang/tests/move_check/expansion/global_access_value.exp

```diff
@@ -3,6 +3,9 @@ error:
    ┌── tests/move_check/expansion/global_access_value.move:3:13 ───
    │
  3 │         1 + ::global_value
-   │             ^^^^^^^^^^^^^^ Unexpected global name '::global_value'. Global names are not values
+   │             ^^^^^^^^^^^^^^ Invalid use of global name '::global_value'
+   ·
+ 3 │         1 + ::global_value
+   │             -------------- Global names can only be used in function calls
```

Comment:
This message seems strange to me. Why show the bad use twice? I think this would be better left as a single message.